src/udiskslinuxblock.c

```diff
@@ -2803,6 +2804,13 @@ udisks_linux_block_is_tcrypt (UDisksBlock *block)
          g_strcmp0 (udisks_block_get_id_type (block), "crypto_TCRYPT") == 0;
 }
 
+gboolean
+udisks_linux_block_is_bitlk (UDisksBlock *block)
+{
+  return g_strcmp0 (udisks_block_get_id_usage (block), "crypto") == 0 &&
+         g_strcmp0 (udisks_block_get_id_type (block), "BitLocker") == 0;
```

Comment:
It's `BitLocker` here because that's what udev/blkid uses, I use `BITLK` in other places because that's what cryptsetup uses. We do the same for TrueCrypt/VeraCrypt where we use `TCRYPT` (again based on cryptsetup).